### PR TITLE
Mark keywords SWATINIT, EQLNUM and SATNUM as IMPORT enabled

### PR DESCRIPTION
**Pull request by joakim-hove**

### CODE REVIEW

**Comment by joakim-hove** [comment]
jenkins build this please

File: src/opm/parser/eclipse/Deck/ImportContainer.cpp [hunk 1]
```diff
@@ -27,8 +27,8 @@
 namespace Opm {
 
 ImportContainer::ImportContainer(const Parser& parser, const UnitSystem& unit_system, const std::string& fname, bool formatted, std::size_t deck_size) {
-    const std::unordered_set<std::string> float_kw = {"COORD", "MULTPV", "NTG", "PERMX", "PERMY", "PERMZ", "PORO", "ZCORN"};
-    const std::unordered_set<std::string> int_kw = {"ACTNUM", "FIPNUM" , "MULTNUM", "FLUXNUM", "OPERNUM"};
+    const std::unordered_set<std::string> float_kw = {"COORD", "MULTPV", "NTG", "PERMX", "PERMY", "PERMZ", "PORO", "SWATINIT, ""ZCORN"};
+    const std::unordered_set<std::string> int_kw = {"ACTNUM", "EQLNUM, ""FIPNUM" , "MULTNUM", "FLUXNUM", "OPERNUM", "SATNUM"};
```
Comment (bska) [inline src/opm/parser/eclipse/Deck/ImportContainer.cpp:31]: Looks like the quoting of `SWATINIT` and `EQLNUM` is invalid here.

Comment (joakim-hove) [reply src/opm/parser/eclipse/Deck/ImportContainer.cpp]: Ehhh - yes; now it works ...

**Comment by joakim-hove** [comment]
jenkins build this please

**Comment by joakim-hove** [comment]
Superceeded by: https://github.com/OPM/opm-common/pull/2417

**Pull request closed without merging**